test/units/modules/network/check_point/test_cp_mgmt_network.py

```diff
@@ -46,13 +46,6 @@
 class TestCheckpointNetwork(object):
```

Comment:
same as in #62213 

`[fedora@ansible  ansible(pull/62214-chkp-orso-second-PR-18-tests)]$ flake8 test/units/modules/network/check_point/test_cp_mgmt_*
test/units/modules/network/check_point/test_cp_mgmt_network.py:22:1: F401 'units.modules.utils.AnsibleFailJson' imported but unused
from units.modules.utils import set_module_args, exit_json, fail_json, AnsibleFailJson, AnsibleExitJson
^
test/units/modules/network/check_point/test_cp_mgmt_network.py:25:1: F401 'ansible.module_utils.network.checkpoint.checkpoint.api_call' imported but unused
from ansible.module_utils.network.checkpoint.checkpoint import api_call
^
`